toolchain/lower/mangler.cpp

```diff
@@ -158,4 +161,17 @@ auto Mangler::Mangle(SemIR::FunctionId function_id,
   return os.TakeStr();
 }
 
+auto Mangler::MangleCppClang(const clang::NamedDecl* decl) -> std::string {
+  if (!cpp_mangle_context_) {
+    // We assume all declarations are from the same AST Context.
+    // TODO: #4666 Consider initializing this in the constructor.
```

Comment:
```suggestion
    // TODO: Consider initializing this in the constructor.
```

It seems like #4666 doesn't talk about this, so maybe it's not useful to mention here?

Note, if you want the connection for some reason, please write it as a sentence with a link (don't assume people know what "4666" means, and make it easy for even the people who know to look in GitHub to get there). For example:

```
    // TODO: Consider initializing this in the constructor. This is related to:
    // https://github.com/carbon-language/carbon-lang/issues/4666
```

Reply:
Yes, I find this link in the code useful to later find relevant TODOs to make sure they're addressed and cleaned up.
Changed the text.